.rubocop.yml

```diff
@@ -18,6 +18,12 @@ AllCops:
     - 'truffleruby/target/classes/**/*'
     - 'mxbuild/**/*'
 
+Lint/LiteralInCondition:
+  Description: Checks of literals used in conditions.
+  Enabled: true
+  Exclude:
+  - lib/truffle/java/java_ext/*.rb
```

Comment:
No point in guessing that it's not much useful, it's enabled by default, it doesn't cause problems, so why would we lose time discussing this and diverge from default?

A case where it could help: If you put `|| true` into an if condition to force a certain branch to try something, this cop will catch it if it's forgotten.